Gom4ziz/Gom4ziz/Presenter/kakaoLoginTest/LoginManagerProtocol.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  LoginManagerProtocol.swift
+//  Gom4ziz
+//
+//  Created by 이가은 on 2022/10/27.
+//
+
+import Foundation
+import RxRelay
+
+enum Loginstatus {
+    case login
+    case logout
+    case unlink
+}
```

Comment:
[수정 고려]
저 여기 한 가지 제안사항이 있어요. LoginStatus 가 유저의 로그인 상태를 나타내는데, login 되있다면 UserInfo가 존재한다는 소리잖아요.  따라서 연관값으로 UserInfo 타입을 가지고 있으면 어떨까요? 즉 로그인 상태와 로그인 데이터가 같이 존재하는거에요.

```suggestion
enum Loginstatus {
    case login(userInfo: UserInfo) // <- 로그인 상태일 땐, 유저인포를 같이 들고 있다. 즉 userInfo 라는 PublishRelay가 따로 필요없어짐
    case logout
    case unlink
}
```

Reply:
수정 제안 감사합니다 ! 이 부분은 후에 @SH0123 과 작업해서 올릴 때 수정해서 올리도록 하겠습니다 !

Reply:
확인했습니다~